GPU/GPUTracking/DataTypes/CalibdEdxContainer.h

```diff
@@ -32,6 +32,22 @@
 namespace o2::tpc
 {
 
+/// flags to set which corrections will be loaded from the CCDB
+enum class Calibsdedx : unsigned short {
```

Comment:
Perhaps
```suggestion
enum class CalibsdEdx : unsigned short {
```
to  be consistent with other names.